catshadow/client_docker_test.go

```diff
@@ -785,22 +817,7 @@ loop7:
 		}
 	}
 	require.Equal(sent, 1)
-
-	if received > 2 {
-		t.Logf("Retransmission of message detected")
-		var last *Message
-		for _, msg := range c {
-			if last == nil {
-				last = msg
-			} else {
-				if bytes.Equal(last.Plaintext, msg.Plaintext) {
-					t.Logf("%s was retransmitted", last.Plaintext)
-				}
-			}
-		}
-	} else {
-		require.Equal(received, 2)
-	}
+	require.Equal(received, 2)
```

Comment:
Wondering if we should assert (for sanity) that:
```golang
require.Equal(1, len(a.conversations))
require.Equal(1, len(b.conversations))
```
It seems like a reasonable requirement, since there is only one conversation going on.

Reply:
Good idea, thanks 923b397542a8323594455f3826ad9019dc6d1791